src/guides/v2.2/extension-dev-guide/plugins.md

```diff
@@ -290,50 +290,132 @@ class MyUtilityUpdater
-| :----------------------: | :--------------: | :----------------------------: | :----------------------------: | :--------------: |
-| **sortOrder**            | 10               | 20                             | 30                             |                  |
-| **before**               | beforeDispatch() | beforeDispatch()               | beforeDispatch()               |                  |
-| **around (first half)**  |                  | aroundDispatch() [first half]  | aroundDispatch() [first half]  |                  |
-| **original**             |                  |                                |                                | dispatch()       |
-| **around (second half)** |                  | aroundDispatch() [second half] | aroundDispatch() [second half] |                  |
-| **after**                | afterDispatch()  | afterDispatch()                | afterDispatch()                |                  |
-| :----------------------: | :--------------: | :----------------------------: | :----------------------------: | :--------------: |
+    * Magento moves on to the next plugin.
 
-The execution flows in the following order:
+* Following the execution flow, starting from lowest to greatest <code>sortOrder</code> in the current sequence plugins loop.
+
+    * The current plugin's `after` method is executed.
+    * Magento moves on to the next plugin.
+
+As a result of these rules, the execution flow of an observed method is affected not only by the prioritization of the plugins but also by their interception implemented methods.  
+
+{:.bs-callout-warning}
+The `around` plugin's method affects the flow of all plugins that are executed after it. 
+
+{:.bs-callout-warning}
+Note that when the `before` and `around` plugin sequence is finished, Magento calls the first plugin `after` method in the sequence loop and not the `after` method of the current plugin that was being executed by the `around` method. 
+
+#### Examples
+For example, the <code>di.xml</code> file of your module attaches three plugins for the class `Action`:
+
+```xml
+<config>
+    <type name="Magento\Framework\App\Action\Action">
+        <plugin name="vendor_module_plugina" type="Vendor\Module\Plugin\PluginA" sortOrder="10" />
+        <plugin name="vendor_module_pluginb" type="Vendor\Module\Plugin\PluginB" sortOrder="20" />
+        <plugin name="vendor_module_pluginc" type="Vendor\Module\Plugin\PluginC" sortOrder="30" />
+    </type>
+</config>
+```
+
+The execution will have a different flow, depending on the methods implemented by these classes, as explained in the following scenarios. 
+
+**Scenario A**
+
+Your plugin classes has this methods: 
+
+|               | PluginA          | PluginB          | PluginC          |
+| :-----------: | :--------------: | :--------------: | :--------------: |
+| **before**    | beforeDispatch() | beforeDispatch() | beforeDispatch() |
```

Comment:
I think the sortOrder row was useful, what about re-introducing it? The same for other tables.